### DISCUSSION TITLE
关于最近蓝灯不能用的问题，有什么解决方法呢？

### DISCUSSION THREAD

**Discussion by casperwzy** [getlantern/lantern-forum-cn#1034] [General]
最近蓝灯基本都不能用，作为已经10年以上的老用户，真的很无语，我还是用的其他的免费VPN才能打开这个留言，真的没办法了，大家有什么好的解决办法吗？

付费的蓝灯能不能有几个稳定的节点，哪怕只能打开网页就行，不至于什么都打不开，不要求速度的，这样可以实现吗？

**Comment by MaxMKMax** [comment]
@casperwzy 您好. 网络不稳定的问题和用的网络提供方有关联. 这个我们这里就控制不了了. 但我会帮您反馈.

**Reply by HIKARU911** [reply]
最近卡顿，我在这里下载了一个139m大小的蓝灯，安装后发现图标变成了瘦长图标，这是正常的吗

**Reply by david-lant** [reply]
请问您在哪里下载的？

**Comment by MaxMKMax** [comment]
@casperwzy 您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。
